doc/esmvalcore/recipe.rst

```diff
@@ -47,27 +58,33 @@ recipe_ocean_amoc.yml.
       projects:
         - ukesm
 
-Note that the authors, projects, and references will need to be included in the
-``config-references.yml`` file. The author name uses the format:
-`surname_name`. For instance, Mickey Mouse would be: `mouse_mickey`.
-Also note that this username is unlikely to be the same as the github
-user name.
+.. note::
 
+   Note that all authors, projects, and references mentioned in the description
+   section of the recipe need to be included in the ``config-references.yml``
+   file. The author name uses the format: ``surname_name``. For instance, John
+   Doe would be: ``doe_john``. This information can be omitted by new users
```

Comment:
Is it not standard to use only four letters of the last name, and two letters of the first name for the authors? So "John Doe" would be "doe_jo"?

Reply:
Yes, but we are trying to get away from it and use full names.
There is already a [PR](https://github.com/ESMValGroup/ESMValTool/pull/1164) for changing this, but new user should add their names using the full names.

Reply:
I see.
Not sure how clear that is with the example provided here where the author name still follows the old standard.

Reply:
`John Doe` --> `doe_john`.